src/main/java/teammates/common/util/Const.java

```diff
@@ -110,6 +110,8 @@
         TIME_REPRESENTS_DEFAULT_TIMESTAMP = TimeHelper.convertToDate("2011-01-01 00:00 AM UTC");
     }
 
+    public static final String TIME_FORMAT_ISO_8601_UTC = "yyyy-MM-dd'T'HH:mm:ss.S'Z'";
```

Comment:
@whipermr5 No worries! :-)  Thanks for including such a detailed explanation for the change and a superb example!